### DISCUSSION TITLE
next-translate is returning key

### DISCUSSION THREAD

**Discussion by ghost** [vinissimus/next-translate#601] [Q&A]
Next translation is displaying key instead of translation

Displayd value common:menu.1.title common:read_more

[https://stackoverflow.com/questions/67322432/next-translate-is-returning-key](url)

Can anyone help?

**Comment by aralroca** [comment]
@aqiros-mculba please share your i18n configuration file, looks that something is incorrect

**Reply by ghost** [reply]
module.exports = {
          locales: ['en', 'de', 'cs'],
          defaultLocale: 'en',
          redirectToDefaultLang: true,
          pages: {
              '*': ['common'],
              '/404': ['home'],
              '/': ['home'],
              '/about': ['about'],
          },
          interpolation: {
              prefix: '${',
              suffix: '}',
          },
          logger: true,
          logBuild: true,
          loadLocaleFrom: (locale, namespace) =>
              import(`./public/locales/${locale}/${namespace}`).then((m) => m.default),
}
